src/os/windows/crashlog_win.cpp

```diff
@@ -454,30 +430,30 @@ char *CrashLogWindows::AppendDecodedStacktrace(char *buffer, const char *last) c
 			}
 
 			/* Print module and instruction pointer. */
-			buffer += seprintf(buffer, last, "[%02d] %-20s " PRINTF_PTR, num, mod_name, frame.AddrPC.Offset);
+			fmt::format_to(output_iterator, "[{:02}] %{:20s} {}", num, mod_name, frame.AddrPC.Offset);
```

Comment:
```suggestion
			fmt::format_to(output_iterator, "[{:02}] {:20s} {}", num, mod_name, frame.AddrPC.Offset);
```
and 3rd param used to be displayed in hexadecimal.

New format:
`[00] %openttd              140700563567662 MainWindow::OnHotkey + 478 (D:\developpement\GitHub\glx22\OpenTTD\src\main_gui.cpp:333)`
Old format:
`[00] openttd              0x00007FF6E08300FE MainWindow::OnHotkey + 478 (D:\developpement\GitHub\glx22\OpenTTD\src\main_gui.cpp:333)`
(removing 0x and leading 0 should be fine)